js/src/index.ts

```diff
@@ -1,2 +1,4 @@
+import * as MltVectorTileJs from './mlt-vector-tile-js/index';
+export { MltVectorTileJs };
```

Comment:
If possible it would be nice to have this package separate so we could test them standalone. I think that would be feasible simply by not integrating into the root index.ts.